### ISSUE TITLE
back camera flipped detection

### ISSUE THREAD

**Issue by streamheart** [Rajatkalsotra/Face-Recognition-with-Flutter#13]
when using the back camera to detect the faces, the bounding box is moving opposite of the subject we are trying to detect

**Comment by Rajatkalsotra** [comment]
Yes, this issue is pending. Camera Preview for the back camera is flipped on the screen

**Comment by IamSubhashish** [comment]
Any solution for the flipped back camera??
